delivery_carrier_label_postlogistics/README.rst

```diff
@@ -23,6 +23,15 @@ As a requirement you need to install `suds-jurko` library. It will fail with the
 latest and outdated version of `suds`.
 https://pypi.python.org/pypi/suds-jurko/0.6
```

Comment:
can you update whole readme based on new layout? https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst